From 0219edad79a1fdfd4d28a9b8998f79135ee89077 Mon Sep 17 00:00:00 2001
From: Ben Hutchings <ben@decadent.org.uk>
Date: Wed, 9 Dec 2015 18:36:04 +0000
Subject: [PATCH 2/2] scripts/functions: Handle PARTLABEL= and PARTUUID= block
 device IDs using blkid

Please can you test this along with the previous patch?

Ben.
---
 scripts/functions | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/functions b/scripts/functions
index d724f74..c7c98c7 100644
--- a/scripts/functions
+++ b/scripts/functions
@@ -295,7 +295,7 @@ resolve_device() {
 	DEV="$1"
 
 	case "$DEV" in
-	LABEL=* | UUID=*)
+	LABEL=* | UUID=* | PARTLABEL=* | PARTUUID=*)
 		if command -v blkid >/dev/null 2>&1; then
 			DEV="$(blkid -l -t "$DEV" -o device)"
 		else
-- 
2.6.2

